lumen/sources/sqlalchemy.py

```diff
@@ -0,0 +1,47 @@
+import param
+
+from sqlalchemy.engine.create import create_engine
+from sqlalchemy.engine.url import URL
+from sqlalchemy.sql.expression import text
+
+from lumen.sources.base import BaseSQLSource
+
+
+class SQLAlchemySource(BaseSQLSource):
+    driver = param.String(default=None, doc="SQL driver.")
```

Comment:
I think it'd be wise to have a way to input URL, perhaps as a classmethod `from_url`

Reply:
+1